docs/testing.md

```diff
@@ -0,0 +1,27 @@
+# Writing Tests
+
+stencil-golang supports three methods of writing tests:
+
+ * unit tests
+ * integration tests (deprecated)
```

Comment:
why even talk about these if we're deprecating them in 14 days apparently

Reply:
Fair, I'll nuke it